skmultilearn/model_selection/iterative_stratification.py

```diff
@@ -103,7 +103,7 @@ def iterative_train_test_split(X, y, test_size, random_state=None):
 
 
 
-def _fold_tie_break(desired_samples_per_fold, M, random_state=None):
+def _fold_tie_break(desired_samples_per_fold, M, rng_state=check_random_state(None)):
```

Comment:
not sure we should deviate from the naming. any reason for that?

Reply:
No, I will fix that, thank you

Reply:
I think that I had something in mind while I was working on it, I changed this to the correct version